build.gradle

```diff
@@ -165,7 +165,7 @@ task deploySite(type: Exec) {
 // Baseline style guide.
 def baselineProjects = [ project("iceberg-api"), project("iceberg-common"), project("iceberg-core"),
```

Comment:
Once `iceberg-pig` and `iceberg-parquet` are fixed. We should remove the baselineProject variable